graphql/tests/myHosts/data.json

```diff
@@ -67,6 +67,73 @@
         "$date": "2020-07-21T10:48:05-04:00"
       }
     },
+    {
+      "_id": "recently-terminated-host",
+      "host_id": "ec2-24-216-756-801.compute-1.amazonaws.com",
+      "user": "ubuntu",
+      "tag": "evg-ubuntu1804-small-20200720144805-8687799505734256781",
+      "distro": {
+        "_id": "ubuntu1804-small",
+        "aliases": ["ubuntu1804"],
+        "arch": "linux_amd64",
+        "work_dir": "/data/mci",
+        "provider": "ec2-ondemand",
+        "user": "ubuntu"
+      },
+      "host_type": "ec2-ondemand",
+      "ext_identifier": "",
+      "display_name": "",
+      "project": "",
+      "zone": "us-east-1a",
+      "provisioned": true,
+      "priv_attempts": 1,
+      "last_task": "",
+      "no_expiration": false,
+      "creation_time": {
+        "$date": "2020-07-20T14:48:05.256Z"
+      },
+      "start_time": {
+        "$date": "2020-07-20T14:48:27Z"
+      },
+      "agent_start_time": {
+        "$date": "2020-07-20T14:49:20.449Z"
+      },
+      "termination_time": {
+        "$date": "5000-01-01T00:00:00Z"
```

Comment:
Usually when we unit test time-dependent functions we can just set the field of the struct to `time.Now()`. But since the data for `graphql` tests comes from a JSON, I couldn't find a way to do that.

To get around this I set this `termination_time` to be in the year 5000, which means that technically it has been terminated in the last 5 minutes, so this host should be returned. I can remove this if you think it's okay to assume that the query works without actually testing it

Reply:
But what happens when we get to the year 5000 our tests will break! JK

But you did reveal a flaw in the way we seed our data that makes it difficult to test features that rely on time. Could you open a ticket so we can look into this and maybe come up with a way to get around this.

Reply:
Opened [EVG-16565](https://jira.mongodb.org/browse/EVG-16565) 🤓